example_14/bringup/config/rrbot_modular_actuators_without_feedback_sensors_for_position_feedback.yaml

```
@@ -1,6 +1,6 @@
 controller_manager:
   ros__parameters:
-    update_rate: 100  # Hz
+    update_rate: 10  # Hz
```

Should we change it back?

```suggestion
    update_rate: 100  # Hz
```